packages/edge-login-ui-react/src/components/ModalAccountCacheDelete.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import { sprintf } from 'sprintf-js'
```

Comment:
Remove 'sprintf' and use built-in string manipulation

Reply:
Actually, we need `sprintf` for localization, since we don't know where in the string the fill-in-the-blank goes. This is the best solution, so let's leave it.